fontdrasil/src/paths.rs

```diff
@@ -1,5 +1,42 @@
 pub fn glyph_file(glyph_name: &str, suffix: &str) -> String {
     // TODO handle names that are invalid for the filesystem
     // Ref https://github.com/unified-font-object/ufo-spec/issues/164
-    glyph_name.to_owned() + suffix
+    let mut filename = Vec::new();
+    for ch in glyph_name.chars() {
+        filename.push(ch);
+
+        // if we wanted reversibility we could append _ when we see an _ in the name
+        // at time of writing we don't want reversibility, merely uniqueness
```

Comment:
you also want to escape the underscore otherwise you may inadvertently confuse two valid glyph names like `Aa` and `a_a`. If both get translated to filename as `a_a` then one may overwrite the other, no?

Reply:
Yes, I think I do!